packages/app-root/src/app/layout.js

```diff
@@ -0,0 +1,19 @@
+import RootLayout from '@/components/RootLayout'
+import StyledComponentsRegistry from './style-registry'
+
+export const metadata = {
+  title: 'Zooniverse',
+  description: 'People-powered Research'
+}
+
+export default function NextLayout({ children }) {
+  return (
+    <html lang='en'>
+      <body style={{ margin: 0 }}>
+        <StyledComponentsRegistry>
+          <RootLayout>{children}</RootLayout>
+        </StyledComponentsRegistry>
+      </body>
```

Comment:
Where does the page `<head>` get built, with the page title, link preview cards etc?

Reply:
Small thing: the page `<head>` is missing the Zooniverse favicon at the moment.

Reply:
Also, all pages have the same metadata at the moment. If I go to http://localhost:3000/users, the page title is still 'Zooniverse' and the description says 'People-powered research'.

Reply:
Yes - I added the need for unique page metadata to the list of next steps for app-root.